ocaml/lambda/translmod.ml

```diff
@@ -164,7 +164,7 @@ and wrap_id_pos_list loc id_pos_list get_field lam =
     List.fold_left (fun (lam, s) (id',pos,c) ->
       if Ident.Set.mem id' fv then
         let id'' = Ident.create_local (Ident.name id') in
-        (Llet(Alias, Lambda.layout_top, id'',
+        (Llet(Alias, Lambda.layout_field, id'',
```

Comment:
I'm not sure it matters, but I think this could be refined to `layout_module_field`.